### PR TITLE
Fix #2035: File formatting under dragAndDropSortInput package

### LINKED ISSUE #2035: Fix file formatting under dragAndDropSortInput package
**Describe the bug**
All the files, under the package `domain/src/test/java/org/oppia/android/domain/classify/rules/dragAndDropSortInput` are not in the correct format. 

**Expected behavior**
File format should be follow from this file - [TextInputEqualsRuleClassifierProviderTest](https://github.com/oppia/oppia-android/blob/develop/domain/src/test/java/org/oppia/android/domain/classify/rules/textinput/TextInputEqualsRuleClassifierProviderTest.kt
)

**Correct Order**
Constant values
Dagger injection
Local variables
Test Before setup
Tests
Dagger Setup

### PR DESCRIPTION
**Pull request by Aarkin7**
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [ ] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [ ] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by Aarkin7** [comment]
Didn't get why did the first two tests failed?

File: domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt [hunk 1]
```diff
@@ -87,4 +85,9 @@ object InteractionObjectTestBuilder {
   fun createInt(value: Int): InteractionObject {
     return InteractionObject.newBuilder().setReal(value.toDouble()).build()
   }
+
+  fun createListOf(vararg items: String): InteractionObject {
+    return InteractionObject.newBuilder().setListOf(items.toString()).build()
+  }
+
```
Comment (Arjupta) [inline domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt:92]: This extra space is causing the lints checks to fail.

File: domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt [hunk 2]
```diff
@@ -19,17 +19,15 @@ object InteractionObjectTestBuilder {
...
-      )
+      .addAllSetOfHtmlStrings(items.map { createHtmlStringList(it) })
       .build()
 
     return InteractionObject.newBuilder().setListOfSetsOfHtmlString(listOfSetsOfHtmlStrings).build()
   }
 
-  fun createHtmlStringList(vararg items: String): StringList {
+  fun createHtmlStringList(vararg items: List<String>): StringList {
     return StringList.newBuilder().addAllHtml(items.toList()).build()
```
Comment (Arjupta) [inline domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt:31]: This changing of datatype to List<String> might be causing the other check to fail. Follow this link to know more about the failed step.
https://github.com/oppia/oppia-android/pull/2053/checks?check_run_id=1320821346#step:9:178

**Review by Arjupta — CHANGES_REQUESTED** [review]
Have commented on some places to check.

**Comment by anandwana001** [comment]
@Aarkin7 Follow our wiki page here for ktlint issues - https://github.com/oppia/oppia-android/wiki/Ktlint-Guide

File: domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt [hunk 3]
```diff
@@ -19,17 +19,15 @@ object InteractionObjectTestBuilder {
     return InteractionObject.newBuilder().setNormalizedString(value).build()
   }
 
-  fun createListOfSetsOfHtmlStrings(listOfStringList: List<StringList>): InteractionObject {
+  fun createListOfSetsOfHtmlStrings(vararg items: List<String>): InteractionObject {
```
Comment (anandwana001) [inline domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt:22]: We don't need to change this.

File: domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt [hunk 4]
```diff
@@ -19,17 +19,15 @@ object InteractionObjectTestBuilder {
...
     val listOfSetsOfHtmlStrings = ListOfSetsOfHtmlStrings.newBuilder()
-      .addAllSetOfHtmlStrings(
-        listOfStringList
-      )
+      .addAllSetOfHtmlStrings(items.map { createHtmlStringList(it) })
```
Comment (anandwana001) [inline domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt:24]: Same here, no modification in this file.

File: domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt (hunk 1 above)
Comment (anandwana001) [reply domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt:92]: Also, from our ktlint wiki page, make sure to setUp macros as per mention in the wiki page.

File: domain/src/test/java/org/oppia/android/domain/classify/rules/dragAndDropSortInput/DragDropSortInputHasElementXAtPositionYRuleClassifierProviderTest.kt [hunk 5]
```diff
@@ -26,11 +24,21 @@ import kotlin.test.fail
...
-  private val STRING_VALUE_3 = createString(value = "test item invalid")
-  private val LIST_OF_SETS_OF_HTML_STRING_VALUE = createListOfSetsOfHtmlStrings()
+
+  private val NON_NEGATIVE_VALUE_TEST_1 =
+    InteractionObjectTestBuilder.createNonNegativeInt(value = 1)
+
+  private val NON_NEGATIVE_VALUE_TEST_2 =
+    InteractionObjectTestBuilder.createNonNegativeInt(value = 2)
+
+  private val STRING_VALUE_TEST_1 =
```
Comment (anandwana001) [inline domain/src/test/java/org/oppia/android/domain/classify/rules/dragAndDropSortInput/DragDropSortInputHasElementXAtPositionYRuleClassifierProviderTest.kt:34]: this should not be `STRING_VALUE_TEST_1`. It should be something related to the test it is going to use for.

**Review by anandwana001 — CHANGES_REQUESTED** [review]
Thanks @Aarkin7 for your contribution. Left some comments for initial pass.

File: domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt (hunk 3 above)
Comment (Aarkin7) [reply domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt:22]: In this function, only one attribute could be passed. I'm getting errors while passing multiple attributes.

Comment (Arjupta) [reply domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt:22]: Tell about the error you are facing

Comment (Aarkin7) [reply domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt:22]: [https://user-images.githubusercontent.com/56022539/97670141-9e26ba00-1aab-11eb-87d7-75d655fe74cc.JPG
](url) 
Error which I'm facing: Too many arguments for public final fun createListOfSetsOfHtmlStrings(listOfStringList: List<[ERROR: StringList]>): [ERROR: InteractionObject] defined in org.oppia.android.domain.classify.InteractionObjectTestBuilder.

Getting this after undoing the changes I did earlier. What changes should I do in fun createListOfSetsOfHtmlStrings in InteractionObjectTestBuilder.kt in order to resolve this?

Comment (Aarkin7) [reply domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt:22]: oppia1
The above mentioned link was not working so please use this one.

Comment (rt4914) [reply domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt:22]: @anandwana001 @ARJUPTA PTAL

Comment (anandwana001) [reply domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt:22]: @Aarkin7 
See the methods argument

```kotlin
createListOfSetsOfHtmlStrings(listOfStringList: List<StringList>)
```

Here, we need to pass a list of `StringList`. 
`StringList` is a custom class here in the codebase. 

To get the string list, we have a method `createHtmlStringList`. List of these `StringList`, you had to pass in this method which asks for a list of `StringList`.

**Comment by Aarkin7** [comment]
@anandwana001 I had created this pr and the previous pr(which has been closed now) from the develop branch of my fork. This could be a reason for generating new pr for the same issue. I have now created a new pr following all the guidelines as mentioned in the wiki page(from a different branch which I have created). So should this pr be closed?

**Comment by Arjupta** [comment]
> @anandwana001 I had created this pr and the previous pr(which has been closed now) from the develop branch of my fork. This could be a reason for generating new pr for the same issue. I have now created a new pr following all the guidelines as mentioned in the wiki page(from a different branch which I have created). So should this pr be closed?

The team might be busy right now so I have seen that new PR as it is solving the same issue then it is okay to close this PR

**Comment by Aarkin7** [comment]
Thanks @ARJUPTA and @anandwana001 for the help. Closing this pr as it violates the guidelines mentioned in the wiki page. I have created a new pr for the same issue.

**Pull request closed without merging**